src/ripple/rpc/handlers/DepositAuthorized.cpp

```diff
@@ -77,6 +77,13 @@ Json::Value doDepositAuthorized (RPC::Context& context)
     if (!ledger)
         return result;
 
+    // If source account is not in the ledger it can't be authorized.
+    if (! ledger->exists (keylet::account(srcAcct)))
+    {
+        RPC::inject_error (rpcSRC_ACT_NOT_FOUND, result);
+        return result;
+    }
+
```

Comment:
The return message for a nonexistent destination account is inconsistent from that of a nonexistent source. Source account error messages seem to have the distinction of `rpcSRC_ACT_MISSING`, meaning the field is absent from the RPC and `rpcSRC_ACT_NOT_FOUND` meaning the object is absent from the ledger. Using a new error code of `rpcDST_ACT_NOT_FOUND` would improve the ambiguity.

Reply:
@miguelportilla, good eyes.  @mDuo13 and I went back and forth a bit about which of the pre-existing error codes to use, but did not discuss creating a new one.  Waddya think, @mDuo13, is it better (from a usability and documentation perspective) to reuse an error code and tolerate some asymmetry or to introduce a new code?  I'm happy either way.